main.go

```diff
@@ -122,11 +122,10 @@ func main() {
 		name := test.SelectAttr("name")
 		status := test.SelectElement("status").SelectAttr("status")
 		suite := test.Parent.SelectAttr("name")
-		startTime := test.SelectElement("status").SelectAttr("starttime")
-		endTime := test.SelectElement("status").SelectAttr("endtime")
```

Comment:
Removing these will then break the duration on RF with version < 7.0. It would be nice to have it backward compatible with older versions also.

Reply:
I don't see a good reason why it has to keep compatibility for RF version earlier 7.0. User still may use the reporter v2.3 for RF prior 7.0.

Reply:
All together users are indirectly forced to upgrade RF to 7.0 in order to use the latest reporter version which is normal for my opinion.

Reply:
Yeah now that you explained it it does make sense. Have to document this also in the action side.